### ISSUE TITLE
Provide win32 exes for Twisted dependencies

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10407]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#513|
|Type|defect|
|Created|2004-02-20 00:55:14Z|

<details><summary>Searchable metadata</summary>

```
trac-id__513 513
type__defect defect
reporter__itamarst itamarst
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1077238514000000 1077238514000000
changetime__1471866530569890 1471866530569890
version__ 
owner__ 
cc__moonfallen cc__radix cc__itamarst
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
I'm doing this as bug since there's no working pyOpenSSL exe installer
publically available anywhere.

We  should provide pyOpenSSL, pySerial, pyCrypto etc. exes, or at least links,
on our download page. For pyOpenSSL we should host on twisted.sf.net so we don't
trash dizzyd's bandwidth:

http://dizzyd.com/pyOpenSSL-0.5.1.win32-py2.2.exe
http://dizzyd.com/pyOpenSSL-0.5.1.win32-py2.3.exe
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
uploaded pyopenssl to twisted.sf.net and added links
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
Added pycrypto 1.9a6 for py2.2 and py2.3 to contrib area of webspace on
twisted.sf.net.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
Added signed MD5SUMS for my pycrypto files.  I suggest rebuilding pyopenssl from
source (after verifying the source's signature) and doing the same with those files.

I guess the other stuff we'll only be linking to, so it's the third party's problem.

FYI: pyserial has binaries for windows on their download page.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
I think this is done?
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_7"></a>|@rodrigc commented|
|-|-|

In changeset 9b3e4d219cd4da8d76fac9461b0ceec528ae5d58
```
#!CommitTicketReference repository="" revision="9b3e4d219cd4da8d76fac9461b0ceec528ae5d58"
Merge pull request #513 from twisted/8776-rodrigc-windowserror-py3

Author: rodrigc
Reviewer: Lukasa, acabhishek942
Fixes: #8776

Fix test_windowsErrorExcept on Windows with Python 3.5
```
